dataset.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""
+The dataset object used in the multi-partner learning and contributivity measurement experiments.
+"""
+
+from sklearn.model_selection import train_test_split
+
+
+class Dataset:
```

Comment:
I would rather call the class Dataset_and_model

Reply:
Ok